app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+    )
+
+
+class Command(BaseCommand):
+    help = """Populates the local test database with dummy data
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--populate',
+            action='store_true',
+            help='Populate dummy data',
+        )
+
+        parser.add_argument(
+            '--clean',
+            action='store_true',
+            help='Clean dummy data',
+        )
+
+    def handle(self, *args, **options):
+
+        geo = Polygon.from_bbox((2570000.0, 1129970.0, 2580000.0, 1139990.0))
+        geo.srid = 2056
+        geo.transform(4326, clone=False)
+        range(2570000, 2746000, 1000)
+        range(1130000, 1252000, 1000)
+
+        if options['clean']:
+            Collection.objects.filter(name__startswith='perftest').delete()
+        elif options['populate']:
+
+            for collection_id in [f'perftest-collection-{x}' for x in range(30)]:
+                collection, _ = Collection.objects.get_or_create(
+                    name=collection_id,
+                    defaults={
+                        'description': 'This is a description',
+                        'license': 'test',
+                        'summaries': {
+                            "eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []
+                        },
+                        'title': 'Test title'
+                    }
+                )
+                for _xmin in range(XMIN, XMAX, 1000):
+                    for _ymin in range(YMIN, YMAX, 1000):
+                        geo = Polygon.from_bbox((_xmin, _ymin, _xmin + 1000, _ymin + 1000))
+                        geo.srid = 2056
+                        geo.transform(4326, clone=False)
+                        item_id = f'perftest-item-{_xmin}-{_ymin}'
+
+                        item, _ = Item.objects.get_or_create(
+                            collection=collection,
+                            name=item_id,
+                            defaults={
+                                'properties_datetime': random_datetime(MIN_DATETIME, MAX_DATETIME),
+                                'properties_title': "My Title",
+                                'geometry': geo
+                            }
+                        )
+
+                        for asset_id in [f'perftest-asset-{x}' for x in range(3)]:
+                            asset, _ = Asset.objects.get_or_create(
+                                item=item,
+                                name=asset_id,
+                                defaults={
+                                    'title': 'my-title',
+                                    'checksum_multihash': "01205c3fd6978a7d0b051efaa4263a09",
+                                    'description': "this an asset",
+                                    'eo_gsd': random.choice([2, 2.5, 5, 10]),
+                                    'geoadmin_lang': random.choice(['de', 'fr', 'it', 'rm', 'en', '']),
```

Comment:
empty language should not be allowed, see https://jira.swisstopo.ch/browse/BGDIINF_SB-1576, either don't generate asset without language or for some asset simply generates the asset without `geoadmin_lang` attribute

Reply:
I'll have to search where we documented it, but for some reason, we intentionally implemented the "empty language" choice. So "empty language" is a valid choice, from what I have in the back of my mind. I'll just need to do some research for the reasoning behind that. (Will do that tomorrow, today I am only working for a very short time..)

Reply:
Probably it was because `geoadmin:lang` is optional. So in the admin interface the dropdown list should provide an "empty" choice as a valid option. But it might make sense to not return empty languages in the API

Reply:
I've found and fixed an issue with the geoadmin:lang in model, still working on it see https://jira.swisstopo.ch/browse/BGDIINF_SB-1576, I'll put a link to this PR as soon as I'm ready for a PR for this issue. But geoadmin:lang empty should definitely not be valid, for optional the field should be simply set to None in model or not added to the payload for the POST method.